src/main/java/gateways/Tasks/TaskResponseModel.java

```diff
@@ -0,0 +1,20 @@
+package gateways.Tasks;
+
+public class TaskResponseModel {
```

Comment:
Having the class name TaskResponseModel is slightly confusing, maybe have it reflect the use case. For instance, CreateTaskResponseModel might be a little more appropriate.

Reply:
I've decided to use one request and response model for Task and Tag. Since I have a significant number of use cases, I feel it would be a lot of redundant code to create request and response models for each. Thus, I have decided to consolidate all the data required for all the use cases into one class.